templates/ingress.yaml

```diff
@@ -20,7 +20,7 @@ metadata:
     kubernetes.io/tls-acme: {{ eq .Values.ingress.acme true | quote }}
     nginx.ingress.kubernetes.io/custom-http-errors: "403,404,502,503"
   {{- if .Values.ingress.auth.enabled }}
-    nginx.ingress.kubernetes.io/auth-url: https://houston.{{ .Values.ingress.baseDomain }}/v1/authorization
+    nginx.ingress.kubernetes.io/auth-url: http://astronomer-houston.{{ .Values.platform.namespace }}.svc.cluster.local:8871/v1/authorization
```

Comment:
@ashb @schnie i have some troubles with this line.

Reply:
I think outside of your local testing this line should need changing at all.

Reply:
Yep, agreed

Reply:
fixed.